doc/man3/SSL_new.pod

```diff
@@ -26,16 +26,9 @@ structure are freed.
 SSL_up_ref() increments the reference count for an
 existing B<SSL> structure.
 
-SSL_dup() duplicates an existing B<SSL> structure into a new allocated one
-or just increments the reference count if the connection is active. All
```

Comment:
We no longer document the "just increments the reference count" behavior but it is still there?

Reply:
It is documented. That aspect was duplicated:

"For connections that are not in their initial state SSL_dup() just increments an internal
reference count and returns the I<same> handle."

Reply:
Oops, I guess I missed that.  Thanks for the clarification, and sorry for the noise.